recursio-client/Global/InputManager.gd

```diff
@@ -71,6 +73,7 @@ func _update_controller_buttons() -> void:
 	# no controller: "keyboard"
 	if controller.size() < 1:
 		_current_controller = "keyboard"
+		Input.set_custom_mouse_cursor(_crosshair_cursor, Input.CURSOR_ARROW, _crosshair_cursor.get_size()/2)
```

Comment:
Maybe add a variable for scaling the crosshair image as a different image would require different scaling here.
Also better to move magic numbers to the top and name them.

Reply:
the "magic number" 2 is for centering the image

Reply:
Oh nvm then